hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/TestAppSchedulingInfo.java

```diff
@@ -175,4 +178,32 @@ public void testSchedulerKeyAccounting() {
     info.updateResourceRequests(reqs, false);
     Assert.assertEquals(0, info.getSchedulerKeys().size());
   }
+
+  @Test
+  public void testApplicationPlacementType() {
```

Comment:
Could you test to verify the behaviour of configured class not available.

Reply:
@bibinchundatt Added UT for case when class is not configured. Please review the PR>